src/modular/monty_form.rs

```diff
@@ -30,6 +31,8 @@ pub struct MontyParams<const LIMBS: usize> {
     /// The lowest limbs of -(MODULUS^-1) mod R
     /// We only need the LSB because during reduction this value is multiplied modulo 2**Limb::BITS.
     mod_neg_inv: Limb,
+    /// Leading zeros in the modulus, used to choose optimized algorithms
+    mod_leading_zeros: u32,
```

Comment:
Hmm, I get the use cases, but this does seem like a potential sharp edge

Reply:
Vartime only!

Reply:
Yeah, I'm just worried someone down the road might be used in constant-time use cases